src/App.vue

```diff
@@ -1,27 +1,69 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png" />
-  <HelloWorld msg="Welcome to Your Vue.js + TypeScript App" />
+  <h1 class="sr-only">Job listings with filtering</h1>
+  <FilterOptions />
+  <Jobs />
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
-import HelloWorld from "./components/HelloWorld.vue";
-
+import Jobs from "./components/jobs/Jobs.vue";
```

Comment:
I recommend you, using aliases for import files.
"@/compoenents/jobs/Jobs.vue"
Because when you move this file, imports will be still OK.